test/cpp/test_cuda_version.cpp

```diff
@@ -0,0 +1,11 @@
+#include <gtest/gtest.h>
+
+#include "../../pyg_lib/csrc/library.h"
+
+TEST(CudaVersionTest, BasicAssertions) {
+#ifdef TEST_WITH_CUDA
+  EXPECT_NE(pyg::cuda_version(), (int64_t)-1);
```

Comment:
Do we really need the `(int64_t)` cast here? Man, I hate C++.

Reply:
Removed it and found it still worked :)